Sources/Interaction/Widgets/LineWidget/index.js

```diff
@@ -141,37 +95,23 @@ function vtkLineWidget(publicAPI, model) {
     publicAPI.setEnable(model.enabled);
   };
 
-  publicAPI.handleMouseMove = () => {
-    publicAPI.moveAction();
-  };
+  publicAPI.handleMouseMove = () => publicAPI.moveAction();
 
-  publicAPI.handleLeftButtonPress = () => {
-    publicAPI.selectAction();
-  };
+  publicAPI.handleLeftButtonPress = () => publicAPI.selectAction();
 
-  publicAPI.handleLeftButtonRelease = () => {
-    publicAPI.endSelectAction();
-  };
+  publicAPI.handleLeftButtonRelease = () => publicAPI.endSelectAction();
 
-  publicAPI.handleMiddleButtonPress = () => {
-    publicAPI.translateAction();
-  };
+  publicAPI.handleMiddleButtonPress = () => publicAPI.translateAction();
 
-  publicAPI.handleMiddleButtonRelease = () => {
-    publicAPI.endSelectAction();
-  };
+  publicAPI.handleMiddleButtonRelease = () => publicAPI.endSelectAction();
 
-  publicAPI.handleRightButtonPress = () => {
-    publicAPI.scaleAction();
-  };
+  publicAPI.handleRightButtonPress = () => publicAPI.scaleAction();
 
-  publicAPI.handleRightButtonRelease = () => {
-    publicAPI.endSelectAction();
-  };
+  publicAPI.handleRightButtonRelease = () => publicAPI.endSelectAction();
 
   publicAPI.selectAction = () => {
     if (model.widgetRep.getInteractionState === InteractionState.OUTSIDE) {
```

Comment:
These are supposed to be calls to getInteractionState, otherwise this will always evaluate to false. The same goes with `publicAPI.translateAction` and `publicAPI.scaleAction`. This isn't related to the PR, but I'm making a note of this as this affects discussion of event abortion in `moveAction`.

Reply:
And actually, fixing this breaks LineWidget.

Reply:
Let's create an issue where we can list problems with the line widget. Thanks @floryst.